### PR TITLE
Use bytestring Builder directly instead of Put

### PR DESCRIPTION
**Pull request by charleso**
/cc @jystic

### CODE REVIEW

File: portmanteau-binary/src/Portmanteau/Binary/VInt.hs [hunk 1]
```diff
@@ -83,10 +83,10 @@ getVInt =
   fromIntegral <$> getVInt64
 {-# INLINE getVInt #-}
 
-putVInt64 :: Int64 -> Put
+putVInt64 :: Int64 -> Builder
 putVInt64 v =
   if v >= -112 && v <= 127 then
-    Binary.putWord8 $ fromIntegral v
+    Binary.singleton $ fromIntegral v
```
Comment (jacobstanley) [inline portmanteau-binary/src/Portmanteau/Binary/VInt.hs]: I think this is wrong, it should be `Builder.word8`

File: portmanteau-binary/src/Portmanteau/Binary/VInt.hs [hunk 2]
```diff
@@ -11,12 +11,12 @@ module Portmanteau.Binary.VInt (
...
   ) where
 
-import           Data.Binary.Builder (Builder)
-import qualified Data.Binary.Builder as Binary
 import           Data.Binary.Get (Get)
 import qualified Data.Binary.Get as Binary
 import           Data.Bits ((.&.), (.|.), complement, shiftL, shiftR, bit)
 import qualified Data.ByteString as B
+import           Data.ByteString.Builder (Builder)
+import qualified Data.ByteString.Builder as Binary
```
Comment (jystic) [inline portmanteau-binary/src/Portmanteau/Binary/VInt.hs]: maybe s/Binary/Builder/ ? /nit

File: portmanteau-binary/src/Portmanteau/Binary/VInt.hs [hunk 3]
```diff
@@ -13,10 +13,10 @@ module Portmanteau.Binary.VInt (
 
 import           Data.Binary.Get (Get)
 import qualified Data.Binary.Get as Binary
-import           Data.Binary.Put (Put)
-import qualified Data.Binary.Put as Binary
 import           Data.Bits ((.&.), (.|.), complement, shiftL, shiftR, bit)
 import qualified Data.ByteString as B
+import           Data.ByteString.Builder (Builder)
+import qualified Data.ByteString.Builder as Binary
```
Comment (jacobstanley) [inline portmanteau-binary/src/Portmanteau/Binary/VInt.hs]: maybe s/as Binary/as Builder/ ? /nit

Comment (charleso) [inline portmanteau-binary/src/Portmanteau/Binary/VInt.hs]: > maybe s/as Binary/as Builder/ 

Fixed

**Comment by jacobstanley** [comment]
Awesome 💯

**Pull request merged**